autogpt_platform/backend/.env.default

```
@@ -302,6 +302,9 @@ TALLY_API_KEY=
 
 # Other Services
 AUTOMOD_API_KEY=
+# All Quiet incident management / on-call. The Public API is only available on
+# the PRO and ENTERPRISE plans; generate the key in your organization settings.
+ALLQUIET_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLQUIET_API_KEY key should go before the AUTOMOD_API_KEY key

(UnorderedKey)